app/precompiled.py

```diff
@@ -723,7 +721,9 @@ def _get_address_from_get_textwords():
         extracted_text.append(" ".join(w[4] for w in gwords))
     extracted_text = "\n".join(extracted_text)
 
-    if rect != NOTIFY_TAG_BOUNDING_BOX and _get_address_from_get_textwords() != extracted_text:
+    if rect != NOTIFY_TAG_BOUNDING_BOX and PrecompiledPostalAddress(
+        _get_address_from_get_textwords()
+    ) != PrecompiledPostalAddress(extracted_text):
```

Comment:
Will call through to `PostalAddress.__eq__` which looks sensible:
https://github.com/alphagov/notifications-utils/blob/4741402d194bb0dfc87d15b2dae16dd193cb5591/notifications_utils/recipient_validation/postal_address.py#L54-L63